extensions/oidc/runtime/src/main/java/io/quarkus/oidc/LogoutConfigBuilder.java

```diff
@@ -0,0 +1,240 @@
+package io.quarkus.oidc;
```

Comment:
Michal, I'd not mind moving all sub-configmapping builders into the `.runtime` package and only keep a top level `OidcTenantConfigBuilder` builder, to allow for some longer term restructurings within those sub-mappings and keep `io.quarkus.oidc` API restricted only to the main `OidcTenantConfig`, does it sound it reasonable ? Otherwise the individual sub-builder may generate a bit of a noise in the API package

Reply:
Well, ideally it would be `OidcTenantConfigBuilder` inner-class, but I was forced 3 largest inner classes outside to limit the `OidcTenantConfigBuilder` size.

> Otherwise the individual sub-builder may generate a bit of a noise in the API package

yeah, I agree about the noise. I can move them to the `runtime`.

Reply:
> moving all sub-configmapping builders

All the `OidcTenantConfigBuilder` inner-classes? at the very least it will require to add getters for every built config group, like if I want to copy `Authentication`, I need to get it first. And I need to copy it if I want to get defaults (or previous values).

TL;DR; I'll change it if you are certain. It is not important IMO, so not an issue to do it either.

Reply:
I want to get this PR done because it has a lot of merge-conflict potential, so I'll make these requested changes right now, won't wait for the confirmation.

Reply:
No, only independent classes like `AuthenticationConfigBuilder`, `LogoutConfigBuilder` and `TenantConfigBuilder`, that should be ok, right ?

Reply:
yes, that makes sense, thanks!